rest/data/commit_queue.go

```diff
@@ -385,15 +385,27 @@ func checkPRApprovals(ctx context.Context, settings *evergreen.Settings, userRep
 		return errors.Wrap(err, "getting GitHub OAuth token from settings")
 	}
 
-	reviews, err := thirdparty.GetGithubPullRequestReviews(ctx, githubToken, userRepo.Owner, userRepo.Repo, prNum)
-	if err != nil {
-		return errors.Wrap(err, "getting GitHub PR reviews")
-	}
-
 	var numApprovals int
-	for _, r := range reviews {
-		if r.GetState() == githubReviewApproved {
-			numApprovals += 1
+	reviewsPage := 0
```

Comment:
nit: the default int is 0, so we can declare this the same as L388 with `var reviewsPage int`.

Reply:
true, fixed